packages/local_auth/README.md

```diff
@@ -130,7 +130,20 @@ app has not been updated to use TouchID.
 Note that local_auth plugin requires the use of a FragmentActivity as
 opposed to Activity. This can be easily done by switching to use
 `FlutterFragmentActivity` as opposed to `FlutterActivity` in your
-manifest (or your own Activity class if you are extending the base class).
+manifest (or your own Activity class if you are extending the base class): 
+
+```java
```

Comment:
The code below is Kotlin, but the marker is Java. Do you think we should add 2 code blocks (covering Java & Kotlin use).